MFVNeutralino/test/TriggerStudies/filtercheck.py

```diff
@@ -1,18 +1,23 @@
 import sys
 from JMTucker.Tools.BasicAnalyzer_cfg import *
+from JMTucker.MFVNeutralino.NtupleCommon import signal_uses_random_pars_modifier
 
 settings = CMSSWSettings()
 settings.is_mc = True
 settings.is_miniaod = True
 settings.cross = '' # 2017to2018' # 2017to2017p8'
 
+randpars_filter = False
+#randpars_filter = 'randpar HToSSTobbbb M15_ct10-'
```

Comment:
We should put this syntax and all of the use cases (that we know of) somewhere!

Reply:
And here it mentions in the next line that it is for submission to condor, while ntuple.py says `# if want to test local : 
`. Does it only work for one vs. the other?

Reply:
Should I create a README in MFVNeutralino/test/ covering these nuances (+ others)  for the random parameter scheme? 
Otherwise, I don't know where all of the possible syntax cases could go without 'being in the way'

Reply:
Seems reasonable to me! We used to have these things tracked in Evernote, but I think a README in the directory is much more sensible. It would actually be _really_ nice to move towards having a README there that also covers important commands, e.g. how to run ntuple.py, histos.py, signal_eff.py, etc.

There is a nearly-blank README in the directory already, so feel free to add to it!